src/Config/Config.ts

```diff
@@ -395,6 +403,7 @@ export class BridgeConfig {
         this.figma = configData.figma;
         this.jira = configData.jira && new BridgeConfigJira(configData.jira);
         this.generic = configData.generic && new BridgeConfigGenericWebhooks(configData.generic);
+        this.feeds = configData.feeds;
```

Comment:
Remember to update the check on line 440

Reply:
Aha, good call. Fixed in 648e457